### DISCUSSION TITLE
Welcome to vanna Discussions!

### DISCUSSION THREAD

**Discussion by zainhoda** [vanna-ai/vanna#192] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sanjaykatheth** [comment]
can we set the time limit in vanna ai

**Comment by wangquan3294** [comment]
i write a code as "from vanna.ollama import Ollama
from vanna.chromadb import ChromaDB_VectorStore" but it tells "  File "C:\Users\DELL\Desktop\testVanna.py", line 2, in <module>
    from vanna.chromadb import ChromaDB_VectorStore
  File "C:\Users\DELL\AppData\Local\Programs\Python\Python312\Lib\site-packages\vanna\chromadb\__init__.py", line 1, in <module>
    from .chromadb_vector import ChromaDB_VectorStore
  File "C:\Users\DELL\AppData\Local\Programs\Python\Python312\Lib\site-packages\vanna\chromadb\chromadb_vector.py", line 4, in <module>
    import chromadb
ModuleNotFoundError: No module named 'chromadb'"
who can help me?
